### DISCUSSION TITLE
Do we have any community calls/slack channel for Triton Inference Server

### DISCUSSION THREAD

**Discussion by vaibhavjainwiz** [triton-inference-server/server#7417] [General]
I am eager to connect with Triton Inference server community and contribute to it. Is there any community meetup call or chat channel available where I could reach out to community.

**Comment by geraldstanje** [comment]
Hi,

I noticed there is no slack, discord or irc channel for tensorrt - which could offload some future tickets by discussing things in the channel - so I created one.

I hope its ok to advertise the discord channel to discuss nvidia tensorrt, tensorrt-llm, triton inference server here: https://discord.gg/hrARsZjJ

cc @vaibhavjainwiz - could you change title - so people see better that we created a discord channel?

Thanks,
Gerald

**Reply by Joejiong** [reply]
not there anymore

**Reply by Pavloveuge** [reply]
@geraldstanje Can you update link, please?
